modules/claims_api/app/controllers/claims_api/base_form_controller.rb

```diff
@@ -76,8 +76,8 @@ def bgs_service
     def intent_to_file_options
       {
         intent_to_file_type_code: ClaimsApi::IntentToFile::ITF_TYPES[form_type],
-        participant_claimant_id: form_attributes['participant_claimant_id'] || target_veteran.participant_id,
-        participant_vet_id: form_attributes['participant_vet_id'] || target_veteran.participant_id,
+        participant_claimant_id: target_veteran.participant_id,
```

Comment:
This needs to be @current_user instead of target user as the claimant is not the veteran but the person making the request on behalf of the the veteran. I would still include form_attributes before calling the || so that if the consumer includes or have a different PID they can do so.